pkg/reconciler/clusterducktype/clusterducktype.go

```diff
@@ -84,12 +86,61 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, dt *v1alpha1.ClusterDuck
 		}
 	}
 
-	dt.Status.Ducks = hunter.Ducks()
+	ducks := hunter.Ducks()
```

Comment:
LOL I forgot I made a hunter package to find Ducks... I am funny.

Reply:
I immediately thought about https://medium.com/everything-80s/the-story-of-duck-hunt-the-original-backbone-of-the-nes-6d4b18c0ff89 when I first saw it :P